app/assets/javascripts/select-kit/addon/components/select-kit/select-kit-header.js

```diff
@@ -1,64 +1,71 @@
 import Component from "@ember/component";
 import { computed } from "@ember/object";
+import {
+  attributeBindings,
+  classNameBindings,
+  classNames,
+} from "@ember-decorators/component";
 import { makeArray } from "discourse-common/lib/helpers";
 import UtilsMixin from "select-kit/mixins/utils";
 
-export default Component.extend(UtilsMixin, {
-  classNames: ["select-kit-header"],
-  classNameBindings: ["isFocused"],
-  attributeBindings: [
-    "role",
-    "tabindex",
-    "selectedValue:data-value",
-    "selectedNames:data-name",
-    "buttonTitle:title",
-    "selectKit.options.autofocus:autofocus",
-  ],
-
-  selectKit: null,
-
-  role: "listbox",
-
-  tabindex: 0,
-
-  selectedValue: computed("value", function () {
+@classNames("select-kit-header")
+@classNameBindings("isFocused")
+@attributeBindings(
+  "role",
+  "tabindex",
+  "selectedValue:data-value",
+  "selectedNames:data-name",
+  "buttonTitle:title",
+  "selectKit.options.autofocus:autofocus"
+)
+export default class SelectKitHeader extends Component.extend(UtilsMixin) {
+  selectKit;
+
+  role = "listbox";
+  tabindex = 0;
```

Comment:
Just for the consistency with other files 🙂 

```suggestion
  selectKit = null;
  role = "listbox";
  tabindex = 0;
```